test/tools/OpenCover/OpenCover.psm1

```diff
@@ -118,43 +124,9 @@ function Get-CoverageSummary([xml.xmlelement] $element)
     }
 
     $CoverageSummary | Add-Member -MemberType ScriptMethod -Name ToString -Value { "Branch:{0,3} Sequence:{1,3}" -f $this.BranchCoverage,$this.SequenceCoverage } -Force
-        
-    return $CoverageSummary
-}
-
-function Expand-ZipArchive([string] $Path, [string] $DestinationPath)
-{
```

Comment:
PSGet supports v4, but isn't there by default.  Not a fan of duplicating expand-archive, but I guess we can keep it for now if expand-archive isn't available.